cli/Sources/TuistServer/Client/ServerAuthenticationController.swift

```diff
@@ -74,92 +88,175 @@ public struct ServerAuthenticationController: ServerAuthenticationControlling {
         #endif
     }
 
+    public func refreshToken(serverURL: URL, inSubprocess: Bool) async throws {
```

Comment:
This method should be available only for the CLI. In other contexts, like macOS or the iOS app, we _can't_ and _shouldn't_ run them in a subprocess.